microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -160,35 +155,51 @@ In the following tables, features are marked with the following statuses:
 === Installation
 * Previously, the `microshift-release-info` RPM depended on the `microshift` RPM. With this release, you can download and install the `microshift-release-info` RPM independently for use in image building. (link:https://issues.redhat.com/browse/OCPBUGS-22854[*OCPBUGS-22854*])
 
-//[discrete]
-//[id="microshift-4-15-storage-bug-fixes"]
-//==== Storage
+* Previously, the Greenboot health check script printed outputs for some checks that were not picked up by `journald`, resulting in missing log entries when running the `journalctl -u greenboot-healthcheck` command. With this release, the production of logs by the Greenboot healthcheck has been fixed so that all outputs are linked to the `systemd` unit, making them easily available to read. (link:https://issues.redhat.com/browse/OCPBUGS-20037[*OCPBUGS-20037*])
 
+* Previously, the Greenboot health check script printed an error message when installed on a non `rpm-ostree` system. The installation was still successful with {microshift-short} starting normally. Now, the error message is not present in the `greenboot-healthcheck` output on non `rpm-ostree`-based systems. (link:https://issues.redhat.com/browse/OCPBUGS-22858[*OCPBUGS-22858*])
 
-//[discrete]
-//[id="microshift-4-15-running-applications-bug-fixes"]
-//==== Running applications
+* Previously, the {microshift-short} clean-up script failed if CRI-O services were not present or not running. This prevented {microshift-short} and its dependencies from fully uninstalling. It also prevented the running of the clean-up script on a system with a new install of the `microshift-rpm` that had not started. With this release, the {microshift-short} clean-up script skips the CRI-O steps if the service is not running, reports that status, and continues with clean-up activities. (link:https://issues.redhat.com/browse/OCPBUGS-22936[*OCPBUGS-22936*])
 
+* Previously, the Greenboot health check reported a `RED` status when the Logical Volume Manager Storage (LVMS) plug-in was disabled because no volume groups (VGs) were present. Because volume groups are not required for {microshift-short}, the health check should report `GREEN` without VGs. Now, when LVMS is disabled, the Greenboot health check skips the check for the `openshift-storage` namespace and reports `GREEN` status. (link:https://issues.redhat.com/browse/OCPBUGS-25689[*OCPBUGS-25689*])
 
-//[discrete]
-//[id="microshift-4-15-support-bug-fixes"]
-//==== Support
+[discrete]
+[id="microshift-4-15-updating-bug-fixes"]
+==== Updating
+* Previously, when updating from one {microshift-short} version to the next on an `rpm-ostree` image, the Greenboot health check failed to read the version file and the update failed. The system rolled back to a healthy state instead of continuing with the update. Now, {microshift-short} correctly handles the legacy format of the internal version file during updates. (link:https://issues.redhat.com/browse/OCPBUGS-19488[*OCPBUGS-19488*])
```

Comment:
I'm not sure why it has target version 4.15, but it was fixes before 4.14 release

Reply:
@pmtk the two bugs are attached to the 4.15.0 Errata. I don't see any other Errata listed in those Jiras, so it doesn't look like those were shipped in a 4.14.z release. I don't see either one in [the async release notes](https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.14/html/release_notes/microshift-4-14-release-notes#microshift-4-14-bug-fixes).